config/docker/security/rotate-secrets.sh

```
@@ -0,0 +1,121 @@
+    openssl genrsa -out "$SECRETS_DIR/ssl_key.pem" 4096
+    chmod 600 "$SECRETS_DIR/ssl_key.pem"
+    
+    # Generate new certificate
+    openssl req -x509 -new -nodes \
+        -key "$SECRETS_DIR/ssl_key.pem" \
+        -sha256 -days 365 \
+        -out "$SECRETS_DIR/ssl_cert.pem" \
+        -subj "/C=US/ST=State/L=City/O=Organization/CN=localhost"
+    
+    # Update Vault if available
+    if [ -n "$VAULT_TOKEN" ] && [ -n "$VAULT_ADDR" ]; then
```

Shellcheck (0.10.0)

[error] Couldn't find 'fi' for this 'if'.

(SC1046)